pkg/resources/database_acceptance_test.go

```diff
@@ -14,8 +15,8 @@ func TestAccDatabase(t *testing.T) {
 		t.Skip("Skipping TestAccDatabase")
 	}
 
-	prefix := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
-	prefix2 := acctest.RandStringFromCharSet(10, acctest.CharSetAlpha)
+	prefix := strings.ToUpper(acctest.RandStringFromCharSet(10, acctest.CharSetAlpha))
```

Comment:
This change is to make the acceptance tests compatible with accounts that have [QUOTED_IDENTIFIERS_IGNORE_CASE](https://docs.snowflake.com/en/sql-reference/parameters.html#quoted-identifiers-ignore-case) set to `true`.  This will cause the any identifiers used to be represented in all upper-case within any system catalog-based queries.

Reply:
That is correct

Reply:
Ok, if this is blocking we can do it for now and I will think of more sustainable ways to do this.